lib/passport-http/strategies/basic.js

```diff
@@ -76,9 +78,10 @@ BasicStrategy.prototype.authenticate = function(req) {
 
   if (!/Basic/i.test(scheme)) { return this.fail(this._challenge()); }
   if (credentials.length < 2) { return this.fail(400); }
-  
+
   var userid = credentials[0];
-  var password = credentials[1];
+  var password = credentials.slice(1).join(':');
```

Comment:
you should use split() with a limit (second parameter), which would avoid the need to workaround the problem in this way.